testing/tools/fix_generator/initial_nbbo_generator/initial_nbbo_generator.pony

```diff
@@ -129,7 +129,9 @@ actor InitialNbboFileGenerator
     _rejected_instruments = rejected_instruments
     _nonrejected_instruments = nonrejected_instruments
     let date = PosixDate(_time._1, _time._2)
-    _utc_timestamp = date.format("%Y%m%d-%H:%M:%S.000")
+    try
+      _utc_timestamp = date.format("%Y%m%d-%H:%M:%S.000")?
```

Comment:
this seems problematic. you are hiding a possible but. I think this should fail if an error gets triggered.

Reply:
updated, now Fails